runtime/caml/bigarray.h

```diff
@@ -34,7 +34,6 @@ typedef unsigned short caml_ba_uint16;
 #define CAML_BA_MAX_NUM_DIMS 16
 
 enum caml_ba_kind {
```

Comment:
It could make sense to have a comment to recommend only appending new enums at the end, to not move tags around for backward compatibility. Alternatively, one could use the explicit form `enum caml_ba_kind { CAML_BA_FLOAT32 = 0, CAML_BA_FLOAT64 = 1, ... }`, which makes it much harder to reorder them without noticing.

Reply:
An alternative would be to add some expect test showing the serialization of bigarrays